web/libs/editor/src/stores/AppStore.js

```diff
@@ -798,11 +798,18 @@ export default types
       self.suggestionsRequest = requestId;
 
       self.setFlags({ awaitingSuggestions: true });
-      const response = yield request;
 
-      if (requestId === self.suggestionsRequest) {
-        self.annotationStore.selected.setSuggestions(dataParser(response));
+      try {
+        const response = yield request;
+
+        if (requestId === self.suggestionsRequest) {
+          self.annotationStore.selected.setSuggestions(dataParser(response));
+          self.setFlags({ awaitingSuggestions: false });
+        }
+      } catch(e) {
         self.setFlags({ awaitingSuggestions: false });
+        // @todo handle errors + situation when task is changed
```

Comment:
since there is a todo here it might be prudent to add a `console.error` temporarily as a placeholder while a more elegant solution is put in place
```suggestion
        // @todo handle errors + situation when task is changed
        console.error('Error:', e);
```

Reply:
state here is not always an error like in this specific case we are fixing here — it's just a way to ignore the wrong request. later we'll come up with two different things:
- proper handling for errors from ML backend
- proper request management and cancellation

Reply:
ok sounds good